http/src/main/resources/init-resources/startup.sh

```diff
@@ -206,13 +206,23 @@ else
         # what was previously under $HOME will now appear in new $HOME as well
         docker exec $JUPYTER_SERVER_NAME /bin/bash -c "[ -d $JUPYTER_USER_HOME/notebooks ] && [ ! -d $JUPYTER_USER_HOME/notebooks/.jupyter ] && rsync -avr --progress --exclude notebooks . $JUPYTER_USER_HOME/notebooks || true"
```

Comment:
Should this line now be moved into the NEEDS_MIGRATE block with a simplified predicate?